drivers/volume/volume.go

```diff
@@ -169,7 +169,7 @@ const (
 )
 
 var (
-	volDrivers = make(map[string]Driver)
+	volDrivers   = make(map[string]Driver)
```

Comment:
Why this change?

Reply:
As aprt of make, it has done formatting and aligned this.